docs/source/user_guide/cli.rst

```diff
@@ -0,0 +1,11 @@
+.. _cli:
+
+======================
+Command Line Interface
+======================
+
+.. automodule:: easylink.cli
+
+.. click:: easylink.cli:easylink
+   :prog: command line
```

Comment:
Is this where the "command line" rather than easylink stuff is coming from?

Reply:
Yep!